automation/Clean.ps1

```diff
@@ -0,0 +1,51 @@
+
+param(
+    [string]$SwitchName = "Default Switch",
+
+    [Parameter(Mandatory=$true)]
+    [string]$VMName,
+
+    [Parameter()]
+    [string]$Platform = "Hyper-V"
+)
+
+if ($VMName) {
+    try{
```

Comment:
We could add a check if the VM exists before everything

Reply:
directory may still exist so I want to continue here